src/FileInfo.cpp

```diff
@@ -87,6 +87,9 @@ FileInfo GetFileInfo(const char *path)
         flags |= FileInfo::kFlagSymlink;
 #endif
 
+    if ((stbuf.st_mode & S_IWRITE) == 0)
```

Comment:
nitpick: would it be better to use S_IWUSR here?

Reply:
That's what I had at first, in fact, but it didn't compile on Windows.